client/cl_property.lua

```diff
@@ -157,10 +157,8 @@ end
 
 function Property:RegisterGarageZone()
     if not next(self.propertyData.garage_data) then return end
-
-    if not (self.has_access or self.owner) or not self.owner then
-        return
-    end
+    
+    if not self.owner or not (self.has_access and Config.AllowAccessToGarage) then return end
```

Comment:
Use the renamed variable `Config.AllowGarageAccessToAllResidents` here and delete Config.AllowAccessToGarage from the config

Reply:
``` if not self.owner or not (self.has_access and Config.AllowGarageAccessToAllResidents) then return end ```

Does not work. Instead it must be changed to:

```
if not (self.has_access and Config.AllowGarageAccessToAllResidents) and not self.owner then
        return
    end
    ```
    
    This was tested thoroughly and is working properly.